diesel_cli/src/query_helper.rs

```diff
@@ -0,0 +1,77 @@
+use diesel::backend::Backend;
+use diesel::query_builder::*;
+use diesel::result::QueryResult;
+
+
+#[derive(Debug, Clone)]
+pub struct DropDatabaseStatement {
+    db_name: String,
+    if_exists: bool,
+}
+
+impl DropDatabaseStatement {
+    pub fn new(db_name: &str) -> Self {
+        DropDatabaseStatement {
+            db_name: db_name.to_owned(),
+            if_exists: false,
+        }
+    }
+
+    pub fn if_exists(mut self) -> Self {
```

Comment:
Unsure how much it'd be inconsistent with the rest of the codebase, but lately I've been preferring this style:

```rust
pub fn if_exists(self) -> Self {
    DropDatabaseStatement { if_exists: true, ..self }
}
```